source/website/src/views/Step4.vue

```diff
@@ -38,8 +38,8 @@ SPDX-License-Identifier: Apache-2.0
             </b-row>
             <b-row>
               <b-col cols="7">
-                <h5>Input file:</h5>
-                {{ "s3://" + DATA_BUCKET_NAME + "/" + s3key }}
+                <h5>Input files from {{"s3://" + DATA_BUCKET_NAME}}:</h5>
```

Comment:
I don't think we should modify the h5 text. That should be kept short because it's a section header.

Reply:
It shouldn't be too long since it's only including the bucket name, but sure I can revert it and have it append the file name to the s3 path like previously `{{ "s3://" + DATA_BUCKET_NAME + "/" + s3key }}` - just was not sure if this seemed redundant if it was all the same bucket